.gitignore

```diff
@@ -3,3 +3,5 @@
 composer.lock
 vendor
 composer.phar
+/nbproject/private/
+nbproject/
```

Comment:
Can we back this change out? It'd be better for you to put in a global ignore file. GitHub has some good directions for this: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore